src/current/_includes/cockroachcloud/filter-tabs/cluster-management.md

```diff
@@ -1,4 +1,4 @@
-{% assign tab_names_html = "CockroachDB Serverless;CockroachDB Dedicated" %}
-{% assign html_page_filenames = "serverless-cluster-management.html;cluster-management.html" %}
+{% assign tab_names_html = "CockroachDB Basic;CockroachDB Standard;CockroachDB Advanced" %}
```

Comment:
We are going back to capitals for the tiers?

Reply:
I assumed we'd follow the convention for the current tiers. I don't remember a discussion about not capitalizing tier names (or I might have missed it)